modules/programs/gnome-terminal.nix

```diff
@@ -30,10 +30,41 @@ let
           description = "The bold color, null to use same as foreground.";
         };
 
+        allowBold = mkOption {
+          default = null;
+          type = types.nullOr types.bool;
+          description = "If true, allow applications in the terminal to make text boldface.";
```

Comment:
Can break the long descriptions over multiple lines to keep them below 80 chars wide. E.g.
```
          description = ''
            If true, allow applications in the terminal to make text boldface.
          '';
```

Similar holds for the other options.